src/index.html

```diff
@@ -16,11 +16,26 @@
         <a href="">Races</a>
     </header>
     <main id="astronaut">
-       
+               
     </main>
+    <div id="overlay-project" class="overlay-project">
+        <div class="works-flex-container-popup displayover">
+            <div class="flexbox-popup">
+                <i class="close fa-solid cancel fa-xmark" onclick="closeDemo()"></i>
+            </div>
```

Comment:
I see the use of onclick attribute directly to call a function closeDemo(), I recommend you use an event listener in your js module to listen to click action, since doing it this way you will be tempted to do that often which can make your code base difficult to maintain and won't follow the KISS principle.